### PR TITLE
New diagonal smooth system and 3/4 walls

### PR DESCRIPTION
**Pull request by Rohesie**
## About The Pull Request

* Ported some 3/4 walls to replace our old top-down perspective ones.
* Made a custom diagonal smoothing system to allow for them. Our current system is very old and doesn't allow for diagonal smoothing, resulting in "holes" in the middle of 4 walls, for example, instead of them being filled with something like a roof.
* Removed the aged turf `max_temperature` var.
* Removed several unused wall icons on the top-down perspective. Will keep adding new ones to have more actual options.
* Fixes a small mapping issue in the Pillar, where the Req shutters didn't have the proper `dir`

## Changelog
:cl:
add: LV and Big Red gain new wall sprites courtesy of the CM dev team, namely jungle, cave and cult walls (the ones in the LV caves).
/:cl:

### CODE REVIEW

**Comment by spookydonut** [comment]
Again, no artist info or explicit artist permission.

**Comment by namjay1170** [comment]
can this please not all be one pr, atleast the map aspect of it

**Comment by Rohesie** [comment]
What would you like to be atomized out of it?

**Comment by namjay1170** [comment]
I'd just like if the map aspect of this PR was its own one

**Comment by Rohesie** [comment]
Sure, I'll do that for the requisitions fix, which can stand on its own.
The theseus and big ERT ship changes, however, are done to remove var-edits and traces of the old smoothing system, and changing that would create conflicts.

**Comment by psykzz** [comment]
We got permission in https://github.com/tgstation/TerraGov-Marine-Corps/pull/4425#issuecomment-658296995 to use these sprites going forward.

**Comment by NethIafin** [comment]
Spriteset is made by Triiodine if we are talking about Big Red and LV. They openly allowed usage of their sprites in OS ahead of CM going OS

**Comment by NethIafin** [comment]
> We got permission in [#4425 (comment)](https://github.com/tgstation/TerraGov-Marine-Corps/pull/4425#issuecomment-658296995) to use these sprites going forward.

wrong spriter ;)

File: code/game/objects/structures/window_frame.dm [hunk 1]
```diff
@@ -9,19 +9,16 @@
...
+	smoothing_groups = SMOOTH_GENERAL_STRUCTURES
 	var/obj/item/stack/sheet/sheet_type = /obj/item/stack/sheet/glass/reinforced
 	var/obj/structure/window/framed/mainship/window_type = /obj/structure/window/framed/mainship
 	var/basestate = "window"
 	var/junction = 0
 	var/reinforced = FALSE
 
-	tiles_with = list(
-		/turf/closed/wall)
+	smoothing_groups = SMOOTH_GENERAL_STRUCTURES
```
Comment (psykzz) [inline code/game/objects/structures/window_frame.dm:20]:
```suggestion
```

defined twice

Comment (Rohesie) [reply code/game/objects/structures/window_frame.dm:20]: This is a depart from the original system and would make them not smooth with anything, not even themselves, no?

Comment (psykzz) [reply code/game/objects/structures/window_frame.dm:20]: No i literally just meant its duplicated from line 13.
image

**Comment by spookydonut** [comment]
Update changelog entry and pr desc with artist info please

**Pull request closed without merging**